doc/api/sqlite.md

```diff
@@ -116,6 +116,8 @@ added: v22.5.0
     and the `loadExtension()` method are enabled.
     You can call `enableLoadExtension(false)` later to disable this feature.
     **Default:** `false`.
+  * `timeout` {number} Number of milliseconds to wait for the database to
```

Comment:
I think we should define this in terms of a busy timeout and link to https://www.sqlite.org/c3ref/busy_timeout.html because I'm not sure everyone will understand what this does.

Reply:
Much better. Will do this.